### DISCUSSION TITLE
Cheat Sheet in Ray

### DISCUSSION THREAD

**Discussion by S-R0** [spatie/ray#449] [Ideas]
Really love using Ray but i've found myself only using a few of the available functions as its really hard to remember all the different functions for each language you use it with.

Some sort of internal docs/cheat sheet would be really handy!

**Comment by freekmurze** [comment]
Here you go: https://spatie.be/docs/ray/v1/usage/reference

**Reply by freekmurze** [reply]
You know what? I'll add a menu item that points to this page

**Reply by S-R0** [reply]
I know there is well written documentation available for Ray already, its what I use at the moment. 

I just meant an in built cheat sheet as an idea/suggestion, as theres so much to remember it would be handy to have it right there if I quickly need to look for something.

Its not a necessity, just an idea for convenience. See the screenshot below of an app I use called **Expressions** where I got the idea from. Its much handier than I initially thought.

![Screenshot 2021-05-07 at 10 27 54](https://user-images.githubusercontent.com/44623107/117431981-bd524880-af21-11eb-9a98-aa6d40cb4a3d.png)
